libbpf-tools/funclatency.c

```diff
@@ -307,6 +307,11 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
+      if (!obj->bss) {
+              warn("BPF_F_MMAPABLE is not supported for arrays\n");
```

Comment:
This message won't help users much. This BPF_F_MMAPABLE feature is available starting from Linux 5.7, so maybe instead print out something like: "Memory-mapping BPF maps is supported starting from Linux 5.7, please upgrade."

Reply:
adjusted.

Reply:
adjusted.